app/helpers/ui_constants.rb

```diff
@@ -176,71 +176,73 @@ module UiConstants
       :vm           => true
     },
     :views => { # List view setting, by resource type
-      :availabilityzone      => "list",
-      :catalog               => "list",
-      :cm_providers          => "list",
-      :cm_configured_systems => "list",
-      :compare               => "expanded",
-      :compare_mode          => "details",
-      :condition             => "list",
-      :container             => "list",
-      :containergroup        => "list",
-      :containernode         => "list",
-      :containerservice      => "list",
-      :containerroute        => "list",
-      :containerproject      => "list",
-      :cimbasestorageextent  => "list",
-      :cimstorageextent      => "list",
-      :cloudtenant           => "list",
-      :drift                 => "expanded",
-      :drift_mode            => "details",
-      :emscloud              => "grid",
-      :emscluster            => "grid",
-      :emscontainer          => "grid",
-      :emsinfra              => "grid",
-      :filesystem            => "list",
-      :flavor                => "list",
-      :host                  => "grid",
-      :job                   => "list",
-      :miqaction             => "list",
-      :miqaeclass            => "list",
-      :miqaeinstance         => "list",
-      :miqevent              => "list",
-      :miqpolicy             => "list",
-      :miqpolicyset          => "list",
-      :miqreportresult       => "list",
-      :miqrequest            => "list",
-      :miqtemplate           => "list",
-      :ontapfileshare        => "list",
-      :ontaplogicaldisk      => "list",
-      :ontapstoragesystem    => "list",
-      :ontapstoragevolume    => "list",
-      :orchestrationstack    => "list",
-      :orchestrationtemplate => "list",
-      :servicetemplate       => "list",
-      :storagemanager        => "list",
-      :miqtask               => "list",
-      :ms                    => "grid",
-      :policy                => "list",
-      :policyset             => "grid",
-      :resourcepool          => "grid",
-      :service               => "grid",
-      :scanhistory           => "list",
-      :snialocalfilesystem   => "list",
-      :storage_files         => "list",
-      :registryitems         => "list",
-      :repository            => "grid",
-      :serverbuild           => "list",
-      :storage               => "grid",
-      :tagging               => "grid",
-      :templatecloud         => "list",
-      :templateinfra         => "list",
-      :treesize              => "20",
-      :vm                    => "grid",
-      :vmcloud               => "grid",
-      :vminfra               => "grid",
-      :vmortemplate          => "grid",
-      :vmcompare             => "compressed",
+      :availabilityzone       => "list",
```

Comment:
@abonas @dclarizio why are these symbols non-delineated?  I would expect `:availabilityzone` to be `:availability_zone` and `:snialocalfilesystem` to be `:snia_local_filesystem`